src/pages/guardian-directory.tsx

```diff
@@ -1,60 +1,108 @@
 import { useEffect, useState } from 'react'
 import { useFlags } from 'launchdarkly-react-client-sdk'
 import { useRouter } from 'next/router'
+import { Button, Search } from '@trussworks/react-uswds'
 import { withDefaultLayout } from 'layout/DefaultLayout/DefaultLayout'
 import styles from 'styles/pages/guardianDirectory.module.scss'
 import { GuardianDirectoryTable } from 'components/GuardianDirectoryTable/GuardianDirectoryTable'
 import { useGetGuardianDirectoryQuery } from 'operations/portal/queries/getGuardianDirectory.g'
+import { useSearchGuardianDirectoryQuery } from 'operations/portal/queries/searchGuardianDirectory.g'
 import { GuardianDirectory as GuardianDirectoryType } from 'types'
-import { useUser } from 'hooks/useUser'
-import Loader from 'components/Loader/Loader'
 
 const GuardianDirectory = () => {
   const flags = useFlags()
   const router = useRouter()
-  const { loading } = useUser()
```

Comment:
I remove useUser and the Loader component because we are already doing this check in the DefaultLayout.